internal/cli/clusters/update.go

```diff
@@ -59,6 +63,84 @@ func (opts *UpdateOpts) initStore(ctx context.Context) func() error {
 }
 
 func (opts *UpdateOpts) Run() error {
+	if opts.isFlexCluster {
+		return opts.RunFlexCluster()
+	}
+
+	return opts.RunDedicatedCluster()
+}
+
+func (opts *UpdateOpts) RunFlexCluster() error {
+	flexClusterReq, err := opts.newFlexCluster()
+	if err != nil {
+		return err
+	}
+
+	flexCluster, err = opts.store.UpdateFlexCluster(opts.ConfigProjectID(), opts.name, flexClusterReq)
+
+	apiError, ok := atlasv2.AsError(err)
+	code := apiError.GetErrorCode()
```

Comment:
Is this safe? I assume that `apiError` will be nil if `ok==false`?

Reply:
That catch me off guard too. Apparently method GetErrorCode handles the nil safety!
```
func (o *ApiError) GetErrorCode() string {
	if o == nil || IsNil(o.ErrorCode) {
		...
```